packages/server/lib/environment.coffee

```diff
@@ -34,6 +34,7 @@ try
 
   if os.platform() is "linux"
     app.disableHardwareAcceleration()
+    app.commandLine.appendSwitch("no-sandbox")
```

Comment:
test to see if this is acceptable vs the CLI argument from `lib/electron`. Let's remove one. Ideally keep this one if both work.

Reply:
needs to exist in lib/electron, or else cypress won't start at all